prisma/schema.prisma

```diff
@@ -875,4 +876,24 @@ model PrismaVeBalTotalSupply {
     chain               Chain
 
     totalSupply         String
-}
\ No newline at end of file
+}
+
+enum PrismaRootStakingGaugeStatus {
+    KILLED
+    ACTIVE
+}
+
+model PrismaRootStakingGauge {
+  @@id([id, chain])
+  @@unique([stakingId, chain])
+
+  id      String
+  chain   Chain
+
+  staking PrismaPoolStakingGauge? @relation(fields: [stakingId, chain], references: [id, chain])
+  status              PrismaRootStakingGaugeStatus @default(ACTIVE)
+  gaugeAddress      String
+  stakingId         String?
+  relativeWeight    String?
+  relativeWeightCap String?
+}
```

Comment:
this needs to be in it's own file in the module. it is merged here by using `yarn prisma-merge`

Reply:
Updated.